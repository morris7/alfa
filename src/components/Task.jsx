import React from 'react';
import { BrowserRouter as Link } from 'react-router-dom';

const styles = {};

const Task = props => {
  const { id, note, name, date } = props;
  return (
    <section>
      <table style={styles}>
        <tbody>
          <tr>
            <td>{id}</td>
            <td>{note}</td>
            <td>{name}</td>
            <td>{date}</td>
          </tr>
          <tr className="taskInfo">
            <td colSpan="4">
              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
              enim ad minim veniam, quis nostrud exercitation ullamco laboris
              nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
              reprehenderit in voluptate velit esse cillum dolore eu fugiat
              nulla pariatur. Excepteur sint occaecat cupidatat non proident,
              sunt in culpa qui officia deserunt mollit anim id est laborum."
            </td>
          </tr>
          <tr>
            <td></td>
            <td></td>
            <td></td>
            <td>
              <Link to="/task">View Task</Link>
            </td>
          </tr>
        </tbody>
      </table>
    </section>
  );
};

export default Task;
