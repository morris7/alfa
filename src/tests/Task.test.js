import React from 'react';
import ReactDOM from 'react-dom';
import Task from '../components/Task';
import { shallow } from 'enzyme';

describe('when rendering the App', () => {
  it('should render without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Task />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('should should match snapshot', () => {
    const wrapper = shallow(<Task />);
    expect(wrapper).toMatchSnapshot();
  });
});
