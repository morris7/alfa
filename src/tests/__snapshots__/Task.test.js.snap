// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when rendering the App should should match snapshot 1`] = `ShallowWrapper {}`;
